util/options_builder.cc

```diff
@@ -7,6 +7,8 @@
 #include <algorithm>
 #include "rocksdb/options.h"
 
+using std::log;
```

Comment:
Removed "using std::log" and "using std::floor" and added "std::" to "log" and "floor" calls.